web/main.js

```diff
@@ -1,164 +1,261 @@
-let metadata, reviews, discussions;
+{
```

Comment:
Wrapped JS logic in a closure as otherwise declarations are made to `window` and can be externally accessed which isn't ideal. Instead this has been done intentionally where needed, e.g. `window.MarkAllAsRead = ...`

However this makes the diff look big, even though it's mostly whitespace. You can change the GitHub diff view to hide whitespace changes.